### DISCUSSION TITLE
Colon emojis in lists

### DISCUSSION THREAD

**Discussion by kamack38** [community/community#18789] [Mobile]
Colon emojis in lists doesn't work on mobile.

**Comment by eliperkins** [comment]
Hey @kamack38, thanks for reaching out! I'm not exactly sure what you mean by colon emojis in lists. Do you have a link to an issue, PR or comment that has an example of this? Or perhaps a screenshot to help my understand a bit better?

I appreciate your help!

**Reply by kamack38** [reply]
By colon emojis I mean emojis like  `:fire:`

**Reply by eliperkins** [reply]
Got it! How about the lists part? Do you mean lists within Markdown comments like:

```markdown
- :fire:
- :100:
```

or are you referring to [GitHub Lists for starred repos](https://docs.github.com/en/get-started/exploring-projects-on-github/saving-repositories-with-stars#organizing-starred-repositories-with-lists)?

**Reply by kamack38** [reply]
I'm referring to GitHub lists for starred repos.

**Reply by kamack38** [reply]
On desktop 'colon emojis' are displayed correctly.

**Reply by francislavoie** [reply]
I'd like to give a +1 for emojis support, would like to see colon emojis in issue/PR labels. We recently added emojis to our labels in https://github.com/caddyserver/caddy/labels but they don't show up on mobile and only display the `:name:` of the emoji.
